knowledge_storm/utils.py

```diff
@@ -33,6 +39,198 @@ def load_api_key(toml_file_path):
 def makeStringRed(message):
     return f"\033[91m {message}\033[00m"
 
+class VectorStoreManager:
```

Comment:
Rename to `QdrantVectorStoreManager`. Also, please add a few comments to describe the functionality of this class and how it can be connected with `VectorRM`.